knowledge-base/chart-draw-horizontal-vertical-lines.md

```diff
@@ -0,0 +1,165 @@
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to draw horizontal lines in Telerik Blazor Charts?
+* How to draw vertical lines in Telerik Blazor Charts?
+* Is it possible to draw vertical or horizontal lines on the Telerik Blazor Chart?
+
+
+## Solution
+
+There are two ways to draw horizontal and vertical lines on the Telerik Blazor Charts.
+
+### Using Plot Bands
+
+Steps for drawing vertical lines with plot bands:
+
+1. Add `ChartXAxisPlotBands` instance inside the `ChartXAxis`.
+2. Add `ChartXAxisPlotBand` instances inside the `ChartXAxisPlotBands` based on the needed number of lines.
+
+Steps for drawing horizontal lines with plot bands:
+
+1. Add `ChartYAxisPlotBands` instance inside the `ChartYAxis`.
+2. Add `ChartYAxisPlotBand` instances inside the `ChartYAxisPlotBands` based on the needed number of lines.
+
+>caption Drawing Horizontal and Vertical lines with Plot Bands
+
+`````CSHTML
+<TelerikChart>
+    <ChartTitle Text="Unrecoverable Errors Per Minute vs. Signal Level"></ChartTitle>
+
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Scatter"
+                     Data="@SeriesData"
+                     Name="APSK modulation"
+                     XField="@nameof(ModelData.Strength)"
+                     YField="@nameof(ModelData.Errors)">
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartXAxes>
+        <ChartXAxis Max="-30" AxisCrossingValue="@(new object[] { -100 })">
+            <ChartXAxisTitle Text="Signal Strength, dBm"></ChartXAxisTitle>
+            <ChartXAxisPlotBands>
+                <ChartXAxisPlotBand From="-90" To="-89.5" Color="blue" />
+                <ChartXAxisPlotBand From="-50" To="-49.5" Color="orange" />
+            </ChartXAxisPlotBands>
+        </ChartXAxis>
+    </ChartXAxes>
+
+    <ChartYAxes>
+        <ChartYAxis Max="20">
+            <ChartYAxisTitle Text="Error count"></ChartYAxisTitle>
+            <ChartYAxisPlotBands>
+                <ChartYAxisPlotBand From="2.5" To="3" Color="green"></ChartYAxisPlotBand>
+                <ChartYAxisPlotBand From="18.5" To="19" Color="red"></ChartYAxisPlotBand>
+            </ChartYAxisPlotBands>
+        </ChartYAxis>
+    </ChartYAxes>
+</TelerikChart>
+
+@code {
+    #region PlotBandsData
+    private List<ModelData> SeriesData { get; set; }
+
+    protected override void OnInitialized()
+    {
+        SeriesData = new List<ModelData>();
+
+        var rnd = new Random();
+
+        for (int i = 0; i < 60; i++)
+        {
+            SeriesData.Add(new ModelData
+                {
+                    Strength = rnd.Next(-90 + i / 3, -90 + i),
+                    Errors = rnd.Next(i / 20, i / 3)
+                });
+        }
+
+        base.OnInitialized();
+    }
+
+    public class ModelData
+    {
+        public double Strength { get; set; }
+        public double Errors { get; set; }
+    }
+    #endregion
+}
+`````
+
+### Using nested Lines Chart
```

Comment:
There is no such thing as "nested" Chart or Series. Think of a better word or just skip it.

```suggestion
### Using Additional Line Series
```